agent/pkg/proxy/iptables.go

```diff
@@ -54,10 +55,7 @@ func newProxier(subnet, netif string, listenIP net.IP, port int) (proxier *Proxi
 		Dst: dst,
 		Gw:  listenIP,
 	}
-	err = netlink.RouteAdd(&proxier.route)
-	if err != nil {
-		klog.Warningf("add route event: %v", err)
-	}
```

Comment:
why delete this code logic, because edgemesh hijacking outbound and inbound traffic from docker0 of this ip range, if you delete this code logic, I can't request service from the host space

Reply:
I will fix this problem